apps/site/assets/package.json

```diff
@@ -17,6 +17,7 @@
     "jquery": "> 3.0.0",
     "leaflet": "^1.4.0",
     "leaflet-rotatedmarker": "^0.2.0",
+    "lodash": "^4.17.11",
```

Comment:
I'm not exactly sure how package-lock is _supposed_ to reflect this change, but I've run the update and committed what it gave me.

Reply:
I had thought since we moved a package from `devDependencies` to `dependencies` that this would be reflected in the lockfile by that dependency flipping from `"dev": true` to `"dev": false`. But since nothing changed (aside from the usual [optional fights](https://github.com/dependabot/feedback/issues/197)), I'm thinking we already had a package in our production `dependencies` that was pulling in Lodash as a dependency of its own, hence it was already marked as `"dev": false`.

tl;dr: we don't have to commit this change to the package-lock 😅